src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,75 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.People;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
+    private static final String MESSAGE_FOR_EXCEPTION = "Can't get min value from list";
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
+                .map(number -> number.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
You can make it easier
```suggestion
                .flatMap(number -> Arrays.stream(number.split(",")))
```

Reply:
done